framework/doc/content/source/preconditioners/HMGPreconditioner.md

```diff
@@ -0,0 +1,65 @@
+# HMG (High-performance (Hybrid) multigrid method)
```

Comment:
This doesn't render very well on our sources table of contents:
<img width="871" alt="Screen Shot 2020-12-02 at 4 03 43 PM" src="https://user-images.githubusercontent.com/10248304/100946387-1112be80-34b8-11eb-9e27-24e53e70a3bf.png">

I would recommend doing the acronym definition within the overview section and just keep the title as HMG. That's consistent with things like `SMP` and `FDP` which are also acronyms

Reply:
Oh, I see. It is too long, haha

Reply:
Great catch

Reply:
Done